Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM ubuntu:20.04
+FROM pytorch/pytorch:latest
 
 ARG PREFIX=/usr/local
 
 # Support multiarch
 RUN dpkg --add-architecture i386
 
 # Add rocm repository
-RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/rocm/apt/5.0.2/ ubuntu main > /etc/apt/sources.list.d/rocm.list'
+RUN sh -c 'echo deb [arch=amd64 trusted=yes] http://repo.radeon.com/rocm/apt/5.2.3/ ubuntu main > /etc/apt/sources.list.d/rocm.list'
```

Comment:
We cannot upgrade to 5.2 in our dockerfile because of issue with our CI.

Reply:
Removed. I saw some odd behavior locally as well.